app-editors/vim/vim-8.2.0639.recipe

```diff
@@ -144,15 +149,8 @@ BUILD()
 INSTALL()
 {
 	make install
-
-	cat << EOF > $dataDir/vim/vimrc
-" Workaround to get the themes and plugins work
-execute "set runtimepath+=" . substitute(system("finddir B_USER_SETTINGS_DIRECTORY"), '\n\+$', '', '') . "/vim"
-EOF
-
-	ln -s vim $binDir/vi
-
-	addAppDeskbarSymlink $binDir/gvim Vim
+	ln -s vim $commandBinDir/vi
+	addAppDeskbarSymlink $commandBinDir/gvim Vim
```

Comment:
As the GUI window shows "VIM" maybe:
```diff
- 	addAppDeskbarSymlink $commandBinDir/gvim Vim
+ 	addAppDeskbarSymlink $commandBinDir/gvim VIM
```

Reply:
The official name is Vim.

Reply:
Doesn't show in the Window :) https://ibb.co/7pmx65Y